packages/loader/driver-utils/src/runWithRetry.ts

```diff
@@ -30,40 +30,77 @@ export interface IProgress {
     /**
      * Called whenever api returns cancellable error and the call is going to be retried.
      * Any exception thrown from this call back result in cancellation of operation
-     * and propagation of thrown exception.
```

Comment:
@vladsud how do you feel about changing the semantics here?  If we want to keep this, I will update "aborted" case to have `reason: unknown` and pass through exceptions thrown from `onRetry` for callers using the old version.

Reply:
Actually I'm looking into just updating all callers and then we don't need 2 versions of `IProgress`